### DISCUSSION TITLE
Browser on RPI: Page starts with an error

### DISCUSSION THREAD

**Discussion by dukeplay** [rern/rAudio#646] [Fixed Issues]
Since the update 20211121 the Chromium starts on my DSI display with this message: "Page Unresponsive. You can wait for it to become responsive or exit the page." and two buttons "wait" or "exit page". With Exit Page I get to another page with Reload. The page will then be restarted and will work until the next reboot.

When I click on Browser on RPI, I get the following message at the end of the parameters:
Nov 22 08:02:57 Teufel sudo[20849]: [20849:20858:1122/080257.126368:ERROR:child_thread_impl.cc(340)] Crashing because hung

![IMG_20211121_101225](https://user-images.githubusercontent.com/92566951/142820832-e09a0e8c-ad51-48f7-bb27-2f612c367996.jpg)
ng

**Comment by rern** [comment]
It's the latest upgrade of Chromium.

Workaround - Downgrade to previous version:
```sh
[[ $( uname -m ) == aarch64 ]] && arch=aarch64 || arch=armv7h
wget http://tardis.tiny-vps.com/aarm/packages/c/chromium/chromium-95.0.4638.69-2-$arch.pkg.tar.xz
pacman -U chromium*
```

**Reply by dukeplay** [reply]
thanks, i did it and it works again!
